primitives/evm-trace/events/src/gasometer.rs

```diff
@@ -30,7 +30,6 @@ impl Snapshot {
 	}
 }
 
-// #[cfg(feature = "evm-tracing")]
```

Comment:
Re-check, remove or un-comment commented?

Reply:
See the comments above.